tool/grind.dart

```diff
@@ -2,13 +2,31 @@
 // MIT-style license that can be found in the LICENSE file or at
 // https://opensource.org/licenses/MIT.
 
+import 'dart:io';
+import 'dart:convert';
+
 import 'package:grinder/grinder.dart';
 
-export 'grind/chocolatey.dart';
-export 'grind/npm.dart';
-export 'grind/github.dart';
-export 'grind/homebrew.dart';
+import 'package:cli_pkg/cli_pkg.dart' as pkg;
+
 export 'grind/sanity_check.dart';
-export 'grind/standalone.dart';
 
-main(List<String> args) => grind(args);
+main(List<String> args) {
+  pkg.humanName = "Sass Migrator";
+  pkg.botName = "Sass Bot";
+  pkg.botEmail = "sass.bot.beep.boop@gmail.com";
+  pkg.executables = {"sass-migrator": "bin/sass_migrator.dart"};
+  pkg.homebrewRepo = "sass/homebrew-sass";
+  pkg.homebrewFormula = "migrator.rb";
+  pkg.jsRequires = {"fs": "fs", "os": "os", "path": "path"};
+  pkg.npmPackageJson =
+      json.decode(File("package/package.json").readAsStringSync())
+          as Map<String, Object>;
```

Comment:
Might it make sense to just move `package.json` to the root so this is autoloaded?

Reply:
Done